### ISSUE TITLE
Chromebook got unstable when opening a folder.

### ISSUE THREAD

**Issue by akimitsuy** [yoichiro/chromeos-filesystem-cifs#162]
My Chromebook got unstable when I tried to open some folders where even just tens of pictures or folders.

When this is happen, the following message is shown at the right bottom of the screen and Chromebook was about to freeze.

'''
操作に予想より時間がかかっています。
中止しますか？
'''

I checked the console log and found that there were a huge access to the samba share, perhaps due to the file app creating the thumbnails of these pictures or folder tree in the folder. 
Console log can't be attached as there were some personal data.

My Chromebook environment is:
ASUS C200MA
Version 46.0.2490.82 (64-bit)
Platform 7390.68.0 (Official Build) stable-channel squawks
Firmware Google_Squawks.5216.152.28

**Comment by yoichiro** [comment]
@akimitsuy Yes, your guess is almost correct. The Files.app creates many events to the FSP implementation depending on files/folders you have. I want to know more detail. Could you provide me the structure of your files/folders simply? For example...

```
/
+ foo/
    - files (x 1,234)
    + bar/                      <- I clicked here!
        - files (x 387)
    + hoge/
    ...
+ baz/
...
```

**Comment by akimitsuy** [comment]
Please find attached [folder_structure.txt](https://github.com/yoichiro/chromeos-filesystem-cifs/files/51632/folder_structure.txt).

For example, when I clicked dir_691 or dir_346, my chromebook got slower, or almost frozen.
The size of files in dir_691 are around 7MB (1.6M pix jpeg) or 18MB (1.6M pix DNG raw).

OTOH, I can browse dir_1 normally.
